protobuf/api/instance.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package api;
+
+service Instance {
+  rpc Create (CreateInstanceRequest) returns (CreateInstanceResponse) {}
+}
+
+message CreateInstanceRequest {
+  string serviceHash = 1;      // Service's sid or hash.
+  repeated string env = 2;     // Env vars to apply to service's instance on runtime.
+}
+
+message CreateInstanceResponse {
+  string hash = 2;        // Service's instance hash.
```

Comment:
@krhubert https://github.com/mesg-foundation/engine/pull/1037#discussion_r291853262
> why starts from 2?

Reply:
because 2 is a good number... just fixed